### PR TITLE
Added support to add oscap capsule to Hostgroup isuue #4171

### LINKED ISSUE #4171: Add oscap capsule support to Hostgroup
Add support to add openscap capsule , while creating hostgroup.

### PR DESCRIPTION
**Pull request by san7ket**
Closes #4171 

Added a test case to show the working of functionality.

### CODE REVIEW

**Comment by san7ket** [comment]
```
 nosetests tests.foreman.ui.test_hostgroup:HostgroupTestCase.test_positive_create_with_oscap_capsule
.
----------------------------------------------------------------------
Ran 1 test in 62.799s

OK
```

File: tests/foreman/ui/test_hostgroup.py [hunk 1]
```diff
@@ -23,10 +23,15 @@
...
 from robottelo.ui.locators import common_locators
 from robottelo.ui.session import Session
+from robottelo.config import settings
 
 
 class HostgroupTestCase(UITestCase):
     """Implements HostGroup tests from UI"""
+    @classmethod
+    def setUpClass(cls):
+        super(HostgroupTestCase, cls).setUpClass()
```
Comment (abalakh) [inline tests/foreman/ui/test_hostgroup.py]: Docstrings are required for all the methods. Please add simple 1-line docstring describing the purpose of defining `setUpClass` here :)

File: tests/foreman/ui/test_hostgroup.py [hunk 2]
```diff
@@ -23,10 +23,15 @@
 from robottelo.ui.factory import make_hostgroup
 from robottelo.ui.locators import common_locators
 from robottelo.ui.session import Session
+from robottelo.config import settings
 
 
 class HostgroupTestCase(UITestCase):
     """Implements HostGroup tests from UI"""
+    @classmethod
```
Comment (abalakh) [inline tests/foreman/ui/test_hostgroup.py]: Please add 1 empty line before this one

File: tests/foreman/ui/test_hostgroup.py [hunk 3]
```diff
@@ -113,3 +120,25 @@ def test_positive_update(self):
...
+    def test_positive_create_with_oscap_capsule(self):
+        """Create new hostgroup with oscap capsule
+
+        @id: c0ab1148-93ff-41d3-93c3-2ff139349884
+
+        @Assert: Hostgroup is created with oscap capsule
+        """
+        name = gen_string('alpha')
+        with Session(self.browser) as session:
+            with self.subTest(name):
```
Comment (ldjebran) [inline tests/foreman/ui/test_hostgroup.py]: why we need a self.subTest here , there is no iteration for name generation

**Comment by san7ket** [comment]
Comments addressed,  please re-review.

**Pull request merged**